man/man8/zpool.8

```diff
@@ -2035,6 +2035,8 @@ output.  See the
 option of
 .Nm zpool Cm iostat
 for complete details.
+.It Fl d
+Display the number of leaf VDEV IO delays.
```

Comment:
Assuming this is talking about `zinject -D`, I think we could improve this documentation, to refer to that feature and to describe the output more precisely.  It isn't clear what "a delay" is, but we could say something like "i/o's are delayed such that they will take at least this many milliseonds to complete".  We should also reference the `zinject` functionality.